tests-utils/src/main/scala/tech.beshu.ror.utils/containers/ReadonlyRestEsClusterContainer.scala

```diff
@@ -185,15 +184,13 @@ class ReadonlyRestEsRemoteClustersContainer private[containers](val localCluster
   }
 
 }
-
-trait ReadonlyRestEsClusterInitializer {
-  def initialize(adminClient: RestClient, container: ReadonlyRestEsClusterContainer): Unit
-}
-
-object NoOpReadonlyRestEsClusterInitializer extends ReadonlyRestEsClusterInitializer {
-  override def initialize(adminClient: RestClient, container: ReadonlyRestEsClusterContainer): Unit = ()
+final case class RorNodeSpecification(envs:Map[String,String])
```

Comment:
what about these ones?
```
xPackSupport: Boolean = false,	                                             
configHotReloadingEnabled: Boolean = true,
customRorIndexName: Option[String] = None,
internodeSslEnabled: Boolean = false
```
at least the first two can also be understood as `RorNodeSpecification` fields

Reply:
I don't see node specification here. I understand node specification, as low level container specification, not business configuration like those.

Reply:
so the name is misleading and should reflect what it actually is

Reply:
I'm not sure if I got you. IMHO setting env variable in my test case shouldn't leak to ror image configuration, but this configuration should be extendable. Additionally extendable configuration should be separated form ROR higher level configs.

Reply:
we could consider several level of settings:
* host/container settings
* jvm settings
* ES (+ RoR) node settings 

`RorNodeSpecification` could suggest that it's related strictly to ROR (no it isn't). It may also suggest that maybe it's a specification of ES node. That's why it's misleading for me.

Moreover, I noticed that in one (or maybe more - to verify) test ENV var is used. It's set like this: https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/blob/master/tests-utils/src/main/scala/tech/beshu/ror/utils/containers/generic/EsImage.scala#L44 Maybe we should remove it and use new approach?

Reply:
Yes, this API can be used. There's so much layers for building. :/ I agree with this settings levels. I think problem was expressing it in `RorNodeSpecification`. I may propose `ContainerSpecification`.

Reply:
Yes, we should (and we will) flat them, but after @pondzix big tests refactoring. 

Please, remove `.env("TEST_VAR", "dev")`. This is used only `DynamicVariablesTests`, so in java ones (@pondzix remember to add it in ContainerSpecification env in rewritten, scala equivalent of `DynamicVariablesTests`)